### PR TITLE
CLI option to write API config to a yaml file

### PR DESCRIPTION
**Pull request by automatthew**
`sky publish staging -o www/_api.yaml`

### CODE REVIEW

File: src/aws/cloudformation.coffee [hunk 1]
```diff
@@ -118,4 +118,12 @@ module.exports = async (env, config) ->
     postDelete = async -> yield src.destroy()
 
 
-    {publish, delete: destroy, publishWait, deleteWait, postPublish, postDelete}
+    {
```
Comment (dyoder) [inline src/aws/cloudformation.coffee]: i guess i understand the use of this, but ideally this is done via discovery, no?

**Comment by automatthew** [comment]
Deferring to the next iteration for proper testing.

File: src/cli.coffee [hunk 2]
```diff
@@ -27,7 +30,14 @@ call ->
...
     .description('deploy API, Lambdas to AWS infrastructure')
-    .action((env)-> run "publish", [env])
+    .option '-o, --output [output]', 'Path to write API config file'
+    .action (env, options) ->
+      call ->
+        stack = yield publish env
+        if options.output?
+          config = url: yield stack.getApiUrl()
+          yield write options.output, (yaml config)
+        console.log bellChar
```
Comment (f1337) [inline src/cli.coffee]: Did you mean to leave this `console.log` dangling?

**Comment by f1337** [comment]
👍 , with the understanding that this is really a hack-around until we get `h9 build` to support env vars.

**Pull request merged**